bigquery/bqml/requirements.txt

```
@@ -1,3 +1,4 @@
-google-cloud-bigquery[pandas]==1.20.0
+pandas==0.22
+google-cloud-bigquery>=1.24.0
```

Can we explicitly pin this? 
```suggestion
google-cloud-bigquery==1.24.0
```